tests/whitebox/integration/test_stratis.py

```diff
@@ -92,11 +92,20 @@ class ErrorHandlingTestCase(SimTestCase):
     Test error-handling behavior when --propagate is not set.
     """
 
-    def testErrorOnMissingPool(self):
+    def testErrorOnMissingFilesystem(self):
         """
-        Test that listing a non-existent pool results in early exit.
+        Test that listing filesystems for a non-existent pool results in early
+        exit.
         """
-        command_line = ["pool", "list", "not_existing"]
+        command_line = ["filesystem", "list", "not_existing"]
+
+        # if exceptions are propagated then a Stratis error is caught
+        with self.assertRaises(StratisCliActionError):
+            RUNNER(["--propagate"] + command_line)
+
+        # If instead the exception chain is handed of to handle_error,
```

Comment:
"If instead the exception chain is handed off to handle_error,"?

Reply:
```handle_error``` is the method that tries to understand the exception chain if the exception is not just propagated.

Reply:
I think it's more a question of if it's off or of. I was wondering if that's a typo too.

Reply:
Oh! I'll fix that typo.